airgun/views/rhai.py

```diff
@@ -35,9 +58,22 @@ def is_displayed(self):
         return self.title.is_displayed and self.search.is_displayed
 
 
+class InventoryHostRule(GenericRemovableWidgetItem):
+    """Insights inventory host rule widget"""
+    # There is no remove button in this widget
```

Comment:
sounds weird. `GenericRemovableWidgetItem` and `There is no remove button in this widget`. `Removable`, but `no remove button`. That is not logical. Any alternative

Reply:
Maybe we can create an issue per our discussion? To create better architecture for such kind of widgets?